spk/vim/Makefile

```diff
@@ -1,6 +1,6 @@
 SPK_NAME = vim
-SPK_VERS = 7.4
-SPK_REV = 2
+SPK_VERS = 8.2
+SPK_REV = 1
```

Comment:
you need to increment revision (SPK_REV) to 3
it must be ascending over all packages with the same name (SPK_NAME) independent of SPK_VERS.
see https://github.com/SynoCommunity/spksrc/wiki/Makefile-variables